src/lib/CSVExport.ts

```diff
@@ -0,0 +1,14 @@
+import jsdownload from "js-file-download";
+import { StaticTableColumns } from "../pages/recentDeliveries/helpers";
+
+export function CSVGenerate(requests) {
+  const csvString = [
+    StaticTableColumns.map(item => item.label),
+    requests.map(req => StaticTableColumns.map(col => req[col.dataKey!]))
+  ]
+
+    .map(e => e.join("\t"))
+    .join("\n");
+
```

Comment:
pending some updates from shalabh to get the export working as desired

additionally, would you mind adding a prompt for the user for where to save the file and name it?